pitch.md

```diff
@@ -1,8 +1,17 @@
-# Project Title (replace me)
+# Construct
 
 ## Overview
 _What's this project about, briefly?_
 
+The Construct language is a domain specific language for doing geometric
+constructions on the fly.
+
+The language aims to
+   1. Allow users to define geometric constructions
+   1. Allow for reuse of constructions.
+   1. Provide an efficient way of determining which constructions may be done
+      next.
+   1. Allow users to render constructions into images or LaTeX.
```

Comment:
How important is this aim relative to the other three? In class you mentioned that you didn't want to focus as much on the output (or input) mechanisms as much as the geometry engine itself.